src/renderer/services/nickname.service.ts

```diff
@@ -0,0 +1,5 @@
+export function adjustNick(nickname: string) {
+    window.electron.ipcRenderer.sendMessage('user:write', [
+        nickname
+    ]);
+}
```

Comment:
It is not very scalable. It is better to have a single channel to read and write config and that the only argument is the entire configuration.